srcpkgs/buildah/template

```diff
@@ -15,7 +15,7 @@ maintainer="Cameron Nemo <cnemo@tutanota.com>"
 license="Apache-2.0"
 homepage="https://github.com/containers/buildah"
 distfiles="${homepage}/archive/v${version}.tar.gz"
-checksum=b53e0a12e6b89b529980b78569c6cea814354aafb65526057c6f67a9924cc3be
+checksum=355b7b6c6486be0375d6ac74c43a61f1bed1faa6c90b1f70c389da4a7fc9a03e
 
 if [ "$XBPS_TARGET_WORDSIZE" = "32" ]; then
 	broken="invalid operation: st.Type == _cgroup2SuperMagic (mismatched types int32 and int64)"
```

Comment:
Is this still an issue?

Reply:
probably not https://github.com/containers/buildah/pull/2100

Reply:
removed